### PR TITLE
AVRO-3488 Fix spelling mistakes

### PR DESCRIPTION
**Pull request by KyleSchoonover**
### Jira

- [ ] My PR addresses the following [Avro Jira](https://issues.apache.org/jira/browse/AVRO/) issues and references them in the PR title. For example, "AVRO-1234: My Avro PR"
  - https://issues.apache.org/jira/browse/AVRO-3488

### Tests

- [ ] My PR does not need testing for this extremely good reason: Non functional changes

### Commits

- [ ] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](https://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: lang/csharp/src/apache/main/IO/BinaryDecoder.netstandard2.0.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public partial class BinaryDecoder
         /// <summary>
         /// A float is written as 4 bytes.
         /// The float is converted into a 32-bit integer using a method equivalent to
-        /// Java's floatToIntBits and then encoded in little-endian format.
+        /// Java's floatToIntBits and then encoded in little-little-endian format.
```
Comment (zcsizmadia) [inline lang/csharp/src/apache/main/IO/BinaryDecoder.netstandard2.0.cs:37]: fix the fix plz :)

**Review by zcsizmadia — CHANGES_REQUESTED** [review]

**Comment by zcsizmadia** [comment]
Nice PR. Mark it for `1.12.0`, in case we are paranoid about the exception text change in a minor release.

File: lang/csharp/src/apache/main/IO/BinaryDecoder.netstandard2.0.cs (hunk 1 above)
Comment (KyleSchoonover) [reply lang/csharp/src/apache/main/IO/BinaryDecoder.netstandard2.0.cs]: There is always one.  Fixed.

**Review by zcsizmadia — APPROVED** [review]

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
Thank you, @KyleSchoonover !

**Pull request merged**